backend/codesirius/problems/views/sample_test.py

```
@@ -0,0 +1,72 @@
+                    data=serializer.data,
+                )
+        except Problem.DoesNotExist:
+            raise ValidationError("Problem not found")
+        except ValidationError as e:
+            raise e
+        except BulkOperationError as e:
+            raise e
+        except ParseError:
+            raise ValidationError({"payload": "Invalid JSON"})
+        except Exception as e:
+            logger.info(f"Type of error: {e.with_traceback()}")
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)